extensions/file/src/test/java/org/mule/extension/file/FileConnectorTestCase.java

```diff
@@ -62,12 +60,6 @@ protected void doTearDownAfterMuleContextDispose() throws Exception
         temporaryFolder.delete();
     }
 
-    @Override
-    protected Class<?>[] getAnnotatedExtensionClasses()
-    {
-        return new Class<?>[] {FileConnector.class};
-    }
-
```

Comment:
Now extensions are auto-discovered by the runner and their metadata is generated while classloader is built.